administrator/language/de-DE/localise.php

```diff
@@ -30,6 +30,9 @@ abstract class De_DELocalise
 	 * @return  array  An array of potential suffixes.
 	 *
 	 * @since   1.6
+	 *
+	 * @deprecated  5.1 will be removed in 7.0 without replacement
```

Comment:
That looks like to be wrong as that method has not been deprecated: https://github.com/joomla/joomla-cms/blob/5.1-dev/libraries/src/Language/Language.php#L355-L371

https://github.com/joomla/joomla-cms/blob/5.1-dev/administrator/language/en-GB/localise.php#L25-L45